storage/innobase/xtrabackup/src/keyring_plugins.cc

```diff
@@ -75,6 +75,7 @@ static void init_plugins(int argc, char **argv) {
 
   mysql_optional_plugins[0] = 0;
 
+  if (!opt_plugin_load_list_ptr->is_empty()) {
```

Comment:
instead of checking not empty. can we check if it contains keyring_file.so to be loaded?

Reply:
@rahulmalik87 In this case we should check for all possible plugins. At the moment only keyring_file.so and keyring_vault. However, we enforce that at load time (we only load keyring%). Do you think it's necessary to repeat the work here?

Reply:
@marce If we ever allow a plugin to be loaded in pxb, we will definitely not know to come and fix this code here. Better to use std::string find with 'keyring'? shouldn't be much overhead

Reply:
@satya-bodapati @rahulmalik87 done.